src/GT-Playground/GTPlayPageFilePersistence.class.st

```diff
@@ -21,6 +26,11 @@ GTPlayPageFilePersistence >> fileName [
 
 { #category : #'accessing-dynamic' }
 GTPlayPageFilePersistence >> fileReference [
+
+	self fileDirectory ifNil: [ 
+		"the directory was not created (read-only filesystem etc.)"
+		^ nil ].
```

Comment:
And special object would avoid such horrible nils

Reply:
A special file reference would be a significant overkill because such then you need a special write stream that does nothing for real too etc. At least, renamed this method to fileReferenceOrNil to explicitly mention the possible special case.